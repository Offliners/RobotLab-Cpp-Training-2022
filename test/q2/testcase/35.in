289
ISOlAtION
170F900
Girl
feature
BALD
SENSATION
TEACH
AmAzE
PENSION
disappoInt
meAnIng
FEW
cHOrUs
PORTIoN
eP60601
kiLLeR
healthy
measles
seveRElY
fork
STArTLE
0Il6N4
sOlitUDe
SLIPpEr
cIViLiaN
CRedItable
Type
FILM
love
contemPLaTE
hurRAh
LOOsEn
OFfER
aGAinsT
InvenTIoN
shelF
GRAVE
reLATiONsHiP
EDGE
dawn
ImAginAtiOn
pArT-TIME
Plot
artilleRy
aIde
DIsTORT
bAgGAGE
obserVaTOry
CAFETERiA
VeGEtaBLE
AfTeRMath
ASSERTION
tWICE
lOyaLTY
IMMEDiATe
TROOP
FOrgive
REgistRaTiON
cHOleSteroL
sevErAL
INtenTion
sAlE
CANe
hUrRY
bEloNg
NOVEMbEr
FRost
swEar
PLANe
EXPENSIVE
tRemendouS
2H546l9
dISPLACeMENT
IMPLEmEnT
multiple
naturAllY
evaluate
CURB
cart
cIVIliZE
M
ENORMOuS
HuMIDIty
serviCe
classical
7u9705
BiLL
HEroIN
dispatch
acCIDEnt
ReCORD
aUtOMObILe/aUTo
VISIBLE
PEnICILlIN
foUNDatIon
TEnSiOn
aQUARIUM
CaRDInAL
mOt9v7te
CRaNE
7ea2
sEcure
piNG-pOng
rEUNION
DESERT
waLkman
paT
Pan
DEteRGeNt
APPlICANt
galLeRy
DEal
dUAl
MObILE
oRIENT
REPuTATION
sTRIdE
FIFTH
PERFECTION
whEElChaIR
reBElLiOus
polluTANT
thus
TumPIKe
tHiRteeN
StRiNG
dIAl
NAG
outline
instiNct
costly
OVERCOAT
3304D7
STICK
shot
tHeAtrical
JuLY
SEveN
Stage
headquarters
HotEl
RELAX
ATtaCh
Pawn
cOmmitteE
FaT
InviTatIon
Us
reDucE
pEst
meDICINE
ANCESTRAL
audience
PatHetic
0IR29LE
deliveR
CLOthEs
sImiLAR
spLenDiD
ChICkEn
QUAR85R
ciRculatIon
GRANdMOTHER/GrAnDMA
OVEReAT
LoOM
CurrIculum
miNUte
bulletin
contrast
DEVICe
temPle
CHILDCaRE
dishwasher
aRouSe
JUNIOR
69C0307
TAll
FEAR
EXpOSE
gigantic
DANCe
chEerful
energY
VERTiCaL
pLAINLy
C
lifetime
CiTiZen
curtain
STAY
rESERVE
bReeder
dISHOneST
BIkiNI
tentative
JeweL
cOURTEsY
LID
SUSpICIoN
hUg
avEnUE
sympathy
roaD
night
STArvE
FORMUlA
nUCLEAR
UnderdeVELopEd
PHilosophy
nAtional
mainlanD
SYLLABLE
seCreTarY
CanDIdate
t52ged2
oCtoBer
8l70
I8Sta77
lEgeND
tunnel
relATE
NAIVE
FIFTEen
ecoNomy
INdiViduAL
log
suffocaTE
CRitic
DESCRIpTION
vIolENCe
controller
ISOLATEd
FAbLE
cRUEL
bury
2mp043aN5
horRiBle
pREgnANt
sPElL
laTItUde
foREcAST
LIVEsTOcK
ATTractiVE
ballAD
TODAY
tendency
office
lock
bowl
suRvIVaL
adJOin
SOleLy
TOOTHPAStE
BEAUTY
murmur
avERaGe
campAign
LARGeLy
vAST
insane
eDUcAted
REPORTER
FUll
2e29L5
SHEPHeRd
cHAIRWOMAN
rAcK
SmilE
souR
tHrEe
kittEn
c
cue
STALL
sPiRITUaL
tRaffic
plEaSe
lOyAl
cD
DetAil
9618U28
dEDiCaTe
uNItY
STAPLE
financE
DRAMATIC
steAM
aBROAD
P
