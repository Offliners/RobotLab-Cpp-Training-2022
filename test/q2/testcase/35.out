n. 隔離
Unknown
n. 少女
n./v. 特徵
adj. 禿的
n. 感覺
v. 教導
v. (n)使驚愕
n. 養老金
v. (n)使失望
n. 意義；重要性
adj./n./pron. 少數
n. 合唱隊
n. 一部分
Unknown
n. 殺人者
adj. 健康的
n. 麻疹
adv. 嚴重地
n./v. 叉子；分岔
v. 使大吃一驚
Unknown
n. 孤獨
n. 拖鞋
n. 平民；文職人員
adj. 值得稱讚的
n. 類型
n./v. 軟片；電影
v./n. 愛；喜歡
v. 深思
interj. 歡呼聲
v. 放鬆
n/v. 提議；提供
prep. 反對；對著
n. 發明；創造力
n. 架子
adj. 重大的；嚴肅的
n. (親戚)關係
n./v. 邊緣；使鋒利
n./v. 黎明；破曉
n. 想像力；創造力
adj./adv. 兼任的
n./v. 情節；圖謀
n. 大砲
n. 助理
adj. 歪曲
n. 行李；裝備
n. 天文臺；氣象臺
n. 自助餐館
n. 蔬菜
n. 餘波
n. 紙張
adv. 兩次
n. 忠誠
adj. 立刻的
n. 部隊
v. 原諒
n. 記錄
n. 膽固醇
adj./n. 幾個的；幾個
n. 意圖；目的
n. 出售；拍賣
n. 藤條；甘蔗
n./v. 急忙
v. 歸屬於
n. ＋一月
n./v. 霜；凍
v. 發誓
n. 飛機
adj. 昂貴的
adj. 極大的
Unknown
n. 移位；取代
v. 實現
adj. 多重的
adv. 自然地
v. 評估
n. 抑制
n./v. 貨車；載運
v. 使開化
Unknown
adj. 巨大的
n. 溼氣
n. 服侍
adj. 極好的；古典的
Unknown
n. 帳單；鈔票
n. 海洛英
v. 派遣；發送
n. 事故
n. 記錄
n. 汽車
adj. 看得見的
n. 盤尼西林
n. 建立；基金會
n. 緊張
n. 水族館
n. 紅衣主教
Unknown
n. 起重機；鶴
Unknown
adj. 安全的
n. 乒乓球
n. 重新結合；聚會
n. 沙漠
n. 隨身聽
n. 拍
n. 平底鍋
n. 洗潔劑
n. 申請人
n. 畫廊；美術館
n ./v. 交易；處理
adj./n. 由兩部分構成的
adj. /n易；汽車
v. 使朝向東方
n. 名聲
v. 邁大步走
adj./n. 第五(的)
n. 完備
n. 輪椅
adj. 難管教的
n. 污染物
adv. 這樣
n. 付費高速公路
adj./n. 十三
n./v. 細繩
v./n. 撥號；刻度盤
v. 不斷地挑剔
n./v. 輪廓；綱要
n. 本能；直覺
adj. 貴重的；代價高的
n. 外套大衣
Unknown
v. 刺入
n. 射擊
adj. 戲劇的
n. 七月
adj./n. 七
n./v. 舞臺；上演
n. 總部
n. 旅館；飯店
v. 放鬆
v.(n. ) 裝上；附加
v. 典當
n. 委員會
adj./n. 肥胖的
n. 邀請
pron. 我們
v. 減少
n. 害蟲
n. 藥；醫學
adj. 祖先的
n. 觀眾；聽眾
adj. 悲哀的
Unknown
v. 遞送
n. 衣服
adj. 相似的
adj. 輝煌的
n. 雞肉；膽小鬼
Unknown
n. 循環；流通；貨幣
n. 祖母；外祖母
v. 暴食
v. 隱約的或威脅性的出現
n. 課程
n. 分鐘
n. 公報；新聞快報
n./v. 對比
n. 裝置
n. 廟宇
n. 照料小孩
n. 洗碗機
v. 喚起
n./adj. 較年少者
Unknown
adj. 高的
n./v. 害怕
v. 使暴露於
adj. 巨人似的；龐大的
n./v. 舞蹈
adj. 興高采烈的
n. 活力
adj. 垂直的
adv. 清楚地
Unknown
n. 一生
n. 公民；市民
n./v. 簾；給…裝上簾子
n./v. 停留
v. 預備
n. 繁殖者
adj. 不誠實的
n. 三點式泳裝
adj. 試驗性的
n. 珠寶
n. 有禮的舉止
n. 蓋子；眼瞼
n. 懷疑
n./v. 擁抱；抱住
n. 大道
n. 同情心
n. 路
n. 夜
v. (使)挨餓
n. 配方；客套話；準則
adj. 核子的
adj. 低度開發的
n. 哲學
adj. 國家的
n. 大陸
n. 音節
n. 秘書
n. 候選人
Unknown
n. 十月
Unknown
Unknown
n. 傳說
n. 隧道
v. 敘述
adj.  天真的；樸素的
adj./n. ＋五
n. 節約；經濟
adj./n. 個別的；個人
n. 原木
v. (使)窒息而死
n. 評論家
n. 描寫；說明?
n. 暴力
n. 控制者
adj. 孤立的
n. 寓言；虛構的故事
adj. 殘酷的
v. 埋
Unknown
adj. 可怕的
adj. 懷孕的
v. 拼寫
n. 緯度
n./v. 預測；預言
n. 家畜
adj. 有吸引力的
n. 民謠
adv./n. 今天
n. 癖好
n. 辦公室
n./v. 鎖
n. 碗
n. 殘存
v. 毗連
adj. 唯一地
n. 牙膏
n. 美人
n./V. 私語
adj./n./v. 平均的
n./v. 競選活動；戰役
adv. 大部分
adj. 廣闊的
adj. 瘋狂的
adj. 受過教育的
n. 報告者
adj. 滿的；完全的
Unknown
n./v. 牧羊人；引導
n. 女主席
n. 置物架
n./v. 微笑
adj. 酸的
adj./n. 三
n. 小貓
Unknown
n. 反芻的食物
n. 攤位
adj. 精神的；心靈的
n. 交通
v. (使)高興
adj. 忠誠的
n. 光碟
n./v. 細節
Unknown
v. 奉獻
n. 統一
n./adj. 釘書針；主要的
n./v. 財政
adj. 戲劇性的
n./v. 蒸汽
adv. 在(到)國外
Unknown
